packages/terra-form-select/src/Combobox.jsx

```diff
@@ -88,10 +88,6 @@ const propTypes = {
    * Callback function invoked for each option on search change. function(searchValue, option)
    */
   optionFilter: PropTypes.func,
-  /**
-   * Placeholder text.
-   */
-  placeholder: PropTypes.string,
```

Comment:
this is breaking change !! we need to update the upgrade guide and change log entry with title `Breaking Change`

Reply:
Updated